assets/js/src/settings/store/make_default_state.ts

```diff
@@ -1,19 +1,59 @@
-import { State } from './types';
+import MailPoet from 'mailpoet';
+import { State, PremiumStatus, MssStatus } from './types';
 import normalizeSettings from './normalize_settings';
 
 export default function makeDefaultState(window: any): State {
+  const pages = window.mailpoet_pages;
+  const segments = window.mailpoet_segments;
+  const save = { inProgress: false, error: null };
+  const data = normalizeSettings(window.mailpoet_settings);
+  const flags = {
+    error: false,
+    woocommerce: !!window.mailpoet_woocommerce_active,
+    newUser: !!window.mailpoet_is_new_user,
+    mssKeyValid: window.mailpoet_mss_key_valid,
+    premiumKeyValid: window.mailpoet_premium_key_valid,
+    premiumPluginInstalled: window.mailpoet_premium_plugin_installed,
```

Comment:
These 3 new flags seem to be used only in this file. What about avoiding adding them to the state? I mean - instead of adding them to flags we can use them just like local variables here (or touch "window" directly).

Reply:
You are right, I missed those.